client/lib/url/support.js

```diff
@@ -18,6 +18,7 @@ export const DOMAINS = `${ root }/domains`;
 export const DOMAIN_CANCEL = `${ root }/domains/#cancel-a-domain-name`;
 export const DOMAIN_EXPIRATION = `${ root }/domains/domain-expiration/`;
 export const DOMAIN_EXPIRATION_REDEMPTION = `${ root }/domains/domain-expiration/#renewing-a-domain-in-the-redemption-period`;
+export const DOMAIN_EXPIRATION_AUCTION = `${ root }/domains/domain-expiration/#expiry-auction`;
```

Comment:
Minor nitpick: This should be defined above `DOMAIN_EXPIRATION_REDEMPTION` to keep the alphabetical order 🤓